cascade-delete.js

```diff
@@ -58,7 +58,17 @@ module.exports = function (Model, options) {
                 var where = {};
                 where[relationKey] = modelId;
 
-                return relationModel.destroyAll(where);
+                if (options.deepDelete && options.deepDelete == true) {
```

Comment:
Looks great. It's good step to create deep delete functionality.
But you can write something like ``` if (options.deepDelete) { ``` It will be enough